src/models/visualisation.py

```diff
@@ -0,0 +1,213 @@
+"""
+This module contains visualisation functions for vector, raster and graph data.
+"""
+
+from typing import Optional, List, Tuple, Callable
+
+import folium
+import geopandas as gpd
+import networkx as nx
+import shapely.geometry
+from src.constants import CHERNOBYL_COORDS_WGS84, UTM35N, CEZ_DATA_PATH
+
+
+def create_graph_visualisation(
+    m: folium.Map = None,
```

Comment:
Minor: I'd suggest refactoring this from `m` to `map` (to stick with naming conventions and be slightly more descriptive)

```suggestion
    map: folium.Map = None,
```